sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -333,7 +334,7 @@ public void beginRecognizeReceiptsFromUrlWithOptions() {
     public void beginRecognizeReceipts() throws IOException {
         // BEGIN: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeReceipts#Flux-long-FormContentType
         File sourceFile = new File("{file_source_url}");
-        Flux<ByteBuffer> buffer = Utility.toFluxByteBuffer(
+        Flux<ByteBuffer> buffer = toFluxByteBuffer(
             new ByteArrayInputStream(Files.readAllBytes(sourceFile.toPath())));
```

Comment:
@mssfang

Following sample code will work:

```java
toFluxByteBuffer(new ByteArrayInputStream(Files.readAllBytes(sourceFile.toPath())));
```

but wondering it should be re-playable to support any retry:

```java
toFluxByteBuffer(Mono.fromCallable(() -> new ByteArrayInputStream(Files.readAllBytes(sourceFile.toPath()))));
```

Not high priority, we can create an issue and link to 9716, we should followup post release, I think scope is bigger here and needs to be discussed in this work item https://github.com/Azure/azure-sdk-for-java/issues/9716